addons/inventory/XEH_preStart.sqf

```diff
@@ -16,6 +16,9 @@ uiNamespace setVariable [QGVAR(ItemKeyCache), _allItems apply {
     if (_picture select [0,1] == "\") then {
         _picture = _picture select [1];
     };
+    if (count _picture > 0 && !(_picture regexMatch ".*?\.paa")) then { // handle missing file extension
+        _picture = _picture + ".paa";
```

Comment:
```suggestion
        if (!fileExists (_picture + ".paa")) exitWith {};
        _picture = _picture + ".paa";
```
not tested, but would something like this help with some edge cases like a mod using png or jpg?

Reply:
Won't hurt, I guess.

Reply:
Should also be pushed to #9696.